.github/dependabot.yml

```diff
@@ -22,5 +22,6 @@ updates:
     open-pull-requests-limit: 200
     reviewers:
       - "vector-im/element-android-reviewers"
-    ignore:
-      - dependency-name: com.google.zxing:core
+    allow:
+      - dependency-name: "io.element.android:wysiwyg"
+      - dependency-name: "org.matrix.rustcomponents:crypto-android"
```

Comment:
I'm not sure this will work as according to [the docs](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#allow)

> a dependency that is matched by both an allow and an ignore will be ignored.

Reply:
Sin I have removed the `ignore` section, I think this will be fine 🤞

Reply:
(because we are in the `package-ecosystem: gradle`)

Reply:
Ah yes, I missed that. Looks good to me then!